lib/Controller/ProxyController.php

```diff
@@ -122,6 +122,10 @@ public function proxy(string $src): ProxyDownloadResponse {
 		$client = $this->clientService->newClient();
 		$response = $client->get($src);
 		$content = $response->getBody();
-		return new ProxyDownloadResponse($content, $src, 'application/octet-stream');
+		$contentType = $response->getHeader('Content-Type');
```

Comment:
but this means that we basically allow *any* content type. If we make this more flexible based on the source than we should have a list of allowed mimes. Anything else should remain `application/octet-stream`

Reply:
How about proxying `image/svg+xml` to fix the svg display issue and serve all other assets with `application/octet-stream`? It might be a good idea to implement proper mime type filtering in a future PR though.

Reply:
Fine by me :)